dataactvalidator/validation_handlers/validator.py

```diff
@@ -782,3 +789,27 @@ def validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType):
         else:
             # Multiple instances of same TAS, something is going wrong
             raise ResponseException("TAS check is malfunctioning",StatusCode.INTERNAL_ERROR)
+
+    @staticmethod
+    def validateFileBySql(submissionId,fileType,interfaces):
+        # Pull all SQL rules for this file type
+        fileId = interfaces.validationDb.getFileId(fileType)
+        rules = interfaces.validationDb.session.query(RuleSql).filter(RuleSql.file_id == fileId).all()
+        errors = []
+        # For each rule, execute sql for rule
+        for rule in rules:
+            failures = interfaces.validationDb.connection.execute(rule.rule_sql.format(submissionId))
+            # Build error list
+            for failure in failures:
+                row = failure["row"]
+                errorMsg = rule.rule_description
```

Comment:
That makes sense, now using rule_error_msg.  Would rule_description still be used for anything?  Even if not, I guess it might still make sense to keep that just so the original description is present somewhere.